include/math.h

```diff
@@ -0,0 +1,10 @@
+#ifndef MATH_H
+#define MATH_H
+
+int add(int x, int y);
```

Comment:
Add doxygen style comments (there is a vscode extension to help) 

```suggestion

/**
 * @brief Adds two integers.
 *
 * This function takes two integer arguments and returns their sum.
 *
 * @param x The first integer to add.
 * @param y The second integer to add.
 * @return The sum of the two integers.
 */
int add(int x, int y);
```

Reply:
This will also auto populate your tooltips in vscode if enabled when you hover over a call of a commented function